packages/atomic/package.json

```diff
@@ -9,7 +9,7 @@
   "types": "dist/types/index.d.ts",
   "collection": "dist/collection/collection-manifest.json",
   "collection:main": "dist/collection/index.js",
-  "unpkg": "dist/atomic/atomic.js",
+  "unpkg": "dist/atomic/atomic.esm.js",
```

Comment:
`dist/atomic/atomic.js` doesn't exist since we aren't building for ES5. Since we're targeting modern browsers, I didn't add the necessary option to build for ES5 in the stencil config, but if you wish I can add it.

Reply:
That should be fine, most browsers today outside IE11 fully support ES6.

Ref: https://caniuse.com/es6